lib/MusicBrainz/Server/Controller/Role/Alias.pm

```diff
@@ -42,9 +44,26 @@ sub aliases : Chained('load') PathPart('aliases')
     my $m = $c->model($self->{model});
     my $aliases = $m->alias->find_by_entity_id($entity->id);
     $m->alias_type->load(@$aliases);
-    $c->stash(
-        aliases => $aliases,
-    );
+
+    if ($entity->entity_type eq 'release_group') {
+        my %props = (
+            aliases => $aliases,
+            canonicalURL => MusicBrainz::Server::Plugin::Canonicalize::replace_gid($self, $c, $entity->gid),
```

Comment:
I don't think this is needed anymore now that we redirect. The MBID in the URL will already be the same as `$entity->gid`.

Reply:
Opened https://github.com/metabrainz/musicbrainz-server/pull/814 to clean that up.